### ISSUE TITLE
Integration with OGC GML tests

### ISSUE THREAD

**Issue by heidivanparys** [INSPIRE-MIF/helpdesk-validator#264]
I really would like to see an integration with the GML tests of OGC. As I have understood it, it was decided not to rely on them as they have issues with large GML files (see e.g. opengeospatial/ets-gml32#22), which is often the case in current set-up in many countries for INSPIRE, especially in the NMCAs.

There is only a textual reference to the OGC tests at https://github.com/inspire-eu-validation/data-encoding/tree/master/inspire-gml:
![image](https://user-images.githubusercontent.com/11427611/77740319-51f63180-7013-11ea-8934-34d1d7f78b7d.png)

However, the OGC tests could still be useful for smaller datasets. A beneficial side-effect would be that the OGC GML tests would be that they would be much wider used, and they could be improved, which would be beneficial for all of us working with GML data. Currently I am experiencing issues with datasets using 3D CRSs - which are more and more used, e.g. for underground infrastructure and city modelling - see e.g. opengeospatial/ets-gml32#45 and opengeospatial/ets-gml32#32.

I am tagging some people here that I think could be interested to participate in this discussion: @dstenger @keshav-nangare @MarcoMinghini @sMorrone

**Comment by danielnavarrogeo** [comment]
Dear @heidivanparys 

Thank you for openning this discussion. Let's wait the feedback from other users.

Regards

**Comment by iuriemaxim** [comment]
@heidivanparys We faced also the problem of validating large datasets and we raised issues both on OGC CITE TeamEngine and on INSPIRE ETF validator. On the other hand there are some bugs in CITE tests that are propagated into the INSPIRE validator. We received an answer from the ETF developers that issues should be
solved in CITE and that other simmilar tests should not be developed in ETF, but only those from CITE should be used. These were in relation to WFS. However now I see that for GML tests there is another vision different from WFS. If for WFS all CITE tests are used even if they have bugs, but for GML, the CITE test are not used (they have bugs as well, some not beeing fixed from 2017). Some references: #208, #215. As a general rule, if the ETF tests are based on CITE tests than it is easy to say that CITE should solve the issue, but till then the service/dataset is not valid (for few years). However if the tests are not depending on CITE, then at least the ETF validator can be fixed, or till a fix is available the test can be very easy swiched off.
